colors.py

```diff
@@ -1,24 +1,27 @@
 """
 This script tries to find the name of color using a given hex value
-List of colors has been obtained from
+List of colors 'color_list.csv' has been obtained from
 https://en.wikipedia.org/wiki/List_of_colors:_A-F
 https://en.wikipedia.org/wiki/List_of_colors:_G-M
 https://en.wikipedia.org/wiki/List_of_colors:_N-Z
 
+List of colors 'xkcd_colors.csv' has been obtained from
+https://xkcd.com/color/rgb/
 """
+col_list = ['color_list.csv', 'xkcd_colors.csv']
 
 colors = dict()
-with open('color_list.csv') as f:
+with open(col_list[1]) as f:
     lis = [line.split(',') for line in f]
     for hex_val, name in lis:
+        if '#' in hex_val:
```

Comment:
Can you have # replaced with 0x in the xkcd_colors.csv so that this operation does not have to be performed everytime the file is read.